darwin/utils.py

```diff
@@ -312,119 +290,108 @@ def parse_darwin_json(path: Path, count: Optional[int]) -> Optional[dt.Annotatio
+        annotation_file = dt.AnnotationFile(
+            path=path,
+            filename=item["name"],
+            annotation_classes=annotation_classes,
+            annotations=annotations,
+            is_video=False,
+            image_width=None,
+            image_height=None,
+            image_url=None,
+            workview_url=item["workview_url"],
+            seq=0,
+            frame_urls=None,
+            remote_path=item["path"],
+        )
+    annotation_file.slots.extend(slots)
+
+    return annotation_file
+
+
+def _parse_darwin_slot(data: Dict[str, Any]) -> dt.Slot:
+    return dt.Slot(
+        name=data["slot_name"],
+        type=data["type"],
+        width=data.get("width"),
+        height=data.get("height"),
+        filename=data.get("file_name"),
+        url=data.get("url"),
+        thubmnail_url=data.get("thumbnail_url"),
+        section_count=data.get("section_count"),
+        section_urls=data.get("section_urls"),
+        fps=data.get("fps"),
     )
 
 
 def _parse_darwin_image(path: Path, data: Dict[str, Any], count: Optional[int]) -> dt.AnnotationFile:
     annotations: List[dt.Annotation] = list(filter(None, map(_parse_darwin_annotation, data["annotations"])))
     annotation_classes: Set[dt.AnnotationClass] = set([annotation.annotation_class for annotation in annotations])
-    return dt.AnnotationFile(
-        path,
-        _get_local_filename(data["image"]),
-        annotation_classes,
-        annotations,
-        False,
-        data["image"].get("width"),
-        data["image"].get("height"),
-        data["image"].get("url"),
-        data["image"].get("workview_url"),
-        data["image"].get("seq", count),
-        None,
-        data["image"].get("path", "/"),
-    )
-
 
-@deprecation.deprecated(
-    deprecated_in="0.7.5",
-    removed_in="0.8.0",
-    current_version=__version__,
-    details="Use 'parse_darwin_json' instead",
-)
-def parse_darwin_video(path: Path, data: Dict[str, Any], count: Optional[int]) -> dt.AnnotationFile:
-    """
-    Parses the given JSON file in v7's darwin proprietary format. Works for playback videos.
-
-    Parameters
-    ----------
-    path : Path
-        Path to the file to parse.
-    data : Dict[str, Any]
-        The decoded JSON file in Python format.
-    count : Optional[int]
-        Optional count parameter. Used only if the data["image"]["seq"] sequence is None.
-
-    Returns
-    -------
-    dt.AnnotationFile
-        An AnnotationFile with the information from the parsed JSON file.
-    """
-
-    annotations: List[dt.VideoAnnotation] = list(filter(None, map(parse_darwin_video_annotation, data["annotations"])))
-    annotation_classes: Set[dt.AnnotationClass] = set([annotation.annotation_class for annotation in annotations])
-
-    if "width" not in data["image"] or "height" not in data["image"]:
-        raise OutdatedDarwinJSONFormat("Missing width/height in video, please re-export")
+    slot = dt.Slot(
+        name="default",
```

Comment:
Well, not really. Everywhere we use "0" as default (I know you prefer "default", but that was a decision made based on discussion with Alberto half a year ago). 
That being said, I'm not sure that matters. This value here is meaningless, there is no way it can be used in any way. Maybe it should be `None` instead to make sure it doesn't get used?